spec/services/course_groups/leadership_spec.rb

```diff
@@ -19,7 +19,9 @@
       end
     end
 
-    context "when date is between December of cohort start year and April of the next year" do
+    context "when date is between December of cohort start year and April of 2026" do
+      let(:cohort) { create(:cohort, start_year: 2025) }
```

Comment:
The context `when` does not match the `let`.
Should the context be `when the cohort is 2025`?
or should there be a before block with the `travel_to` in it?
Not a blocker for merging, just a nice to have.

Reply:
I'd argue no - this is for the 2025 cohort but spring 2026 schedule _that being said_ I'll put the names back to approximately how they were to avoid confusion